setup.cfg

```diff
@@ -4,7 +4,7 @@ version = 0.0.1
 description = Exersices for INF200
 author = Elin Bjoernson
 author_email = elinbj@nmbu.no
-
+    
```

Comment:
Please remove any trailing white-space that makes files without actual changes are added in the pull requests.

Reply:
I think I have removed trailing white space now